File: app/src/main/java/com/example/realestateassignment/model/Exclusion.java
package com.example.realestateassignment.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Exclusion {

    @SerializedName("facility_id")
    @Expose
    private String facilityId;
    @SerializedName("options_id")
    @Expose
    private String optionsId;

    public String getFacilityId() {
        return facilityId;
    }

    public void setFacilityId(String facilityId) {
        this.facilityId = facilityId;
    }

    public String getOptionsId() {
        return optionsId;
    }

    public void setOptionsId(String optionsId) {
        this.optionsId = optionsId;
    }

}

File: app/src/main/java/com/example/realestateassignment/model/Exclusion.kt
package com.example.realestateassignment.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Exclusion {
    @SerializedName("facility_id")
    @Expose
    var facilityId: String? = null

    @SerializedName("options_id")
    @Expose
    var optionsId: String? = null
}